table.csv

```diff
@@ -124,6 +124,7 @@ http,                           multiaddr,      0x01e0,         draft,
 swhid-1-snp,                    ipld,           0x01f0,         draft,     SoftWare Heritage persistent IDentifier version 1 snapshot
 json,                           ipld,           0x0200,         permanent, JSON (UTF-8-encoded)
 messagepack,                    serialization,  0x0201,         draft,     MessagePack
+car,                            serialization,  0x0202,         draft,     Content Addressable aRchive (CAR)
```

Comment:
can we specify this is a carv1 specifically, or does this cover both car v1 and car v2?

Reply:
Clarify as in a table somehow or here ? If here it's supposed to be version agnostic.